DevelopedCalculator.js

```diff
@@ -0,0 +1,9 @@
+class DevelopedCalculator extends Calculator {
+    render_() {
```

Comment:
The name of this method must bee `render` otherwise when you are doing this
`developedCalculator.render()`
the parent class render is been invoked not `DevelopedCalculator` render method.

you need to have a single interface but in side the new render method call the parent render method